README.adoc

```diff
@@ -1,68 +1,205 @@
 ifndef::env-github[:icons: font]
 ifdef::env-github[]
 :outfilesuffix: .adoc
 :caution-caption: :fire:
 :important-caption: :exclamation:
-:note-caption: :paperclip:
+:note-caption: :information_source:
 :tip-caption: :bulb:
 :warning-caption: :warning:
 endif::[]
-
+// Custom attributes
+:chromium-browser-list: Chrome, Edge, Chromium, Brave or Opera
+:project-name: Asciidoctor Browser Extension
+:uri-project-repo: https://github.com/asciidoctor/asciidoctor-browser-extension
 :uri-license: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/LICENSE
-:uri-hacking: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/hacking.adoc
+:uri-contributing: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/CONTRIBUTING.adoc
 :uri-changelog: https://github.com/asciidoctor/asciidoctor-chrome-extension/blob/main/changelog.adoc
+// Webstores
 :uri-chrome-webstore: https://chrome.google.com/webstore/detail/asciidoctorjs-live-previe/iaalpfgpbocpdfblpnhhgllgbdbchmia
 :uri-mozilla-addon: https://addons.mozilla.org/firefox/addon/asciidoctorjs-live-preview
 :uri-opera-addon: https://addons.opera.com/extensions/details/asciidoctorjs-live-preview
 :uri-edge-webstore: https://microsoftedge.microsoft.com/addons/detail/asciidoctorjs-live-previ/pefkelkanablhjdekgdahplkccnbdggd
+// Webstore graphics
+:uri-chrome-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/chromewebstore-badge-wborder-496x150.png
+:uri-mozilla-addon-image: https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg
+:uri-opera-addon-image: https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png
+:uri-edge-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png
 
-This project uses https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] to preview AsciiDoc as HTML inside Chrome (or Chromium), Firefox and Opera!
+Enjoy AsciiDoc everywhere your browser goes with the {project-name} (the Extension).
 
-image:https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/chromewebstore-badge-wborder-496x150.png[link={uri-chrome-webstore},height="64px",width="212px"]
-image:https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg[link={uri-mozilla-addon},height="64px",width="184px"]
-image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png[link={uri-opera-addon},height="64px",width="227px"]
-image:https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png[link={uri-edge-webstore},height="64px",width="177px"]
+The {project-name} uses https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] to preview AsciiDoc files (.ad, .adoc, .asc, .asciidoc, and optionally .txt) as HTML inside Chrome (or Chromium), Brave, Firefox, Opera and Microsoft Edge.
```

Comment:
(suggestion): I think we should extract `https://github.com/asciidoctor/asciidoctor.js` as an attribute

Reply:
Done.